### DISCUSSION TITLE
Error during template rendering

### DISCUSSION THREAD

**Discussion by brightmorkli37** [reactive-python/reactpy-django#150] [Problem] [answered]
Following the outlined procedure in the outstanding documentation gives me the following error;

**TemplateSyntaxError at /
'reactpy' is not a registered tag library. Must be one of:
......**

Did I miss a step?

**Comment by Archmonger** [comment, marked as answer]
You most likely forgot to add `reactpy_django` to installed apps.

See the [installation docs](https://reactive-python.github.io/reactpy-django/get-started/installation/#step-2-configure-settingspy).
